### ISSUE TITLE
Nano quantization default value result is incorrect

### ISSUE THREAD

**Issue by yangw1234** [intel-analytics/BigDL#7480]
The following test would fail

![image](https://user-images.githubusercontent.com/10561966/217654277-446b64dc-99eb-45d3-8726-d8c19c1f5da4.png)

reproduce:

https://github.com/intel-analytics/BigDL/pull/7479

**Comment by rnwang04** [comment]
I have tried to reproduce this code, but I don't get any error.
```python
import torch
from bigdl.nano.pytorch import InferenceOptimizer
from torch import nn
import numpy as np

def test_onnx_inc_default_values():
    # default bool values
    class Net(nn.Module):
        def __init__(self):
            super().__init__()

        def forward(self, x, a=None):
            if a is None:
                return x
            else:
                return x+1

    model = Net()

    data = torch.zeros(1,3,1,1)
    print(data)
    result_true = model(data)
    print(result_true)
    # sample with only required parameters (in a tuple)
    accmodel = InferenceOptimizer.quantize(model,
                                            accelerator="onnxruntime",
                                            calib_data=torch.rand(2,3,1,1))
    result_m = accmodel(data)
    print(result_m)
    assert abs(torch.sum(result_m).item()) < 1e-5
```
As for https://github.com/intel-analytics/BigDL/pull/7479, if we change `torch.zeros(1,3,1,1)` to `torch.zeros(1,3,1,1) +- number`, `assert abs(torch.sum(result_m).item()) < 1e-5` must fail as it is a normal behaviour.
@TheaperDeng please correct me if there is any problem.

**Comment by yangw1234** [comment]
@rnwang04 
There are two problems here.

1. https://github.com/intel-analytics/BigDL/actions/runs/4128517969/jobs/7133058678
   This is the code you tried. (maybe using INC 2.0)
   I think it is fixed on latest master.
2. https://github.com/intel-analytics/BigDL/actions/runs/4128517969/jobs/7133058522
   This is another failed job of #7479, using INC 1.13 and pytorch 1.12 
   In this case, it failed on the first assertion, where `data = torch.zeros() - 1` and model should go the `return x + 1` branch.
   I do not think it is fixed on latest master. The test is skipped when inc is less than 2.0.
   
![image](https://user-images.githubusercontent.com/10561966/218872420-655d514a-47bf-4bee-87f1-58cb51955795.png)

**Comment by rnwang04** [comment]
@yangw1234 
1. yeah, it's fixed on latest master.
2.  I have reproduce this problem with INC 1.13 and pytorch 1.12. **It can be easily fixed by upgrade INC to 1.14 and larger version.**
I think it's a bug of INC 1.13 itself. However, we can't upgrade INC to 1.14 in related workflow as other uts (related to pytorch lightning) will fail (as a known issue).
So we may just update the ut (https://github.com/intel-analytics/BigDL/pull/7553) and make this as a known issue.
